config.js

```diff
@@ -41,7 +41,8 @@ module.exports = {
     returnsGracePeriod: process.env.RETURNS_GRACE_PERIOD || 21,
     // The date SROC goes live
     isSrocLive,
-    srocStartDate
+    srocStartDate,
+    alcsEndYear: 2022
```

Comment:
no need for configuration from an ENV ?

Reply:
Yeah I guess it could have gone in the env but if you look at the config file all our feature flag and configurations are there. ENV is mainly for secrets and keys